chromadb/utils/query_helper.py

```diff
@@ -0,0 +1,135 @@
+from typing import List, Union, Literal, Any, Dict, cast
```

Comment:
I think we should make the API able to consume these rather than expecting users to call .to_filter(). So the API methods check if the input is one of these, compile it and then proceed.